Dockerfile

```diff
@@ -21,10 +20,14 @@ RUN apt-get update \
   && apt-get -y install nodejs npm \
   && apt-get clean
 
+# This is required because there has been an upgrade of this package which hasn't yet been updated in the base image
+# This can be removed when the base image is updated with the upgrade
+RUN apt-get install --only-upgrade libexpat1
```

Comment:
Getting an error locally when run_localing.

 => ERROR [web base  6/16] RUN apt-get install --only-upgrade libexpat1    1.0s 
------                                                                          
 > [web base  6/16] RUN apt-get install --only-upgrade libexpat1:               
0.144 Reading package lists...                                                  
0.670 Building dependency tree...                                               
0.811 Reading state information...                                              
0.985 The following additional packages will be installed:
0.985   libexpat1-dev
1.008 The following packages will be upgraded:
1.008   libexpat1 libexpat1-dev
1.016 2 upgraded, 0 newly installed, 0 to remove and 2 not upgraded.
1.016 Need to get 249 kB of archives.
1.016 After this operation, 3072 B disk space will be freed.
1.016 Do you want to continue? [Y/n] Abort.
------
failed to solve: process "/bin/sh -c apt-get install --only-upgrade libexpat1" did not complete successfully: exit code: 1

Didn't prune everything yet but let me know if I should

Reply:
Separately do we need to remove hyphen between docker and compose in run_local?

Reply:
fixed, thanks